wrapper-cpp/c_meeting_service_interface.cpp

```
@@ -74,7 +75,7 @@ extern "C" ZOOMSDK::SDKError meeting_join(
     param.meetingNumber = mid;
```

Clang (14.0.6)

[warning] narrowing conversion from 'unsigned long' to signed type 'UINT64' (aka 'int') is implementation-defined

(bugprone-narrowing-conversions,cppcoreguidelines-narrowing-conversions)